lib/src/user.dart

```diff
@@ -172,7 +172,8 @@ class User extends Event {
   /// Whether the client is allowed to change the power level of this user.
   /// Please be aware that you can only set the power level to at least your own!
   bool get canChangePowerLevel =>
-      room.canChangePowerLevel && powerLevel < room.ownPowerLevel;
+      (room.canChangePowerLevel && powerLevel < room.ownPowerLevel) ||
```

Comment:
I have no idea what this function is supposed to do, how is it different from room.canChangePowerLevel, a rename would be nice.

Also `powerLevel < room.ownPowerLevel` does not make sense